core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/shared/ui/dot-experiments-inline-edit-text/dot-experiments-inline-edit-text.component.html

```diff
@@ -0,0 +1,53 @@
+<p-inplace>
+    <ng-template pTemplate="display">
+        <div class="flex">
+            <span *ngIf="text; else noTextDefinedTpl" data-testId="text-input">
+                {{ text }}
+            </span>
+            <i class="pi pi-pencil cursor-pointer" data-testId="text-input-icon"></i>
+        </div>
+
+        <ng-template #noTextDefinedTpl
+            ><span class="text-muted" data-testId="empty-text-message">{{
+                'experiments.configure.description.add' | dm
```

Comment:
add this as input to make it more generic ?

Reply:
Every inplace in the system looks different :'(

Reply:
Added